### PR TITLE
feat(client): integrate comments apis with frontend components.

### PR DESCRIPTION
**Pull request by piyushagwl007**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#147 

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by piyushagwl007** [comment]
AddComment

File: client/src/components/Comment/CommentComponent.jsx [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
+  commentId,
+   image,
+   author,
+   postedOn,
+  content,
+  blogId,
+  _id,
+  username,
 }) => {
+  const isTempComment = commentId.indexOf('temp') > -1;
```
Comment (nihalanisumit) [inline client/src/components/Comment/CommentComponent.jsx:21]: what is tempCpomment?

File: client/src/components/Comment/index.jsx [hunk 2]
```diff
@@ -1,41 +1,63 @@
...
   const [formInput, setFormInput] = useState('');
-  const [comments, setComment] = useState([]);
-  const author = 'Jon Snow';
+  const [comments, updateComments, newComment, addNewComment] = useComments(blogId);
 
   const createComment = (event, commentContent) => {
     event.preventDefault();
     setFormInput('');
-    if (!commentContent || commentContent.trim() === '') {
+    if (!commentContent || commentContent.trim() === '' || !_id) {
```
Comment (nihalanisumit) [inline client/src/components/Comment/index.jsx]: explain this?

File: client/src/utils/date/fromNow.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import moment from 'moment';
+
+const fromNow = (dateString) => moment(dateString).fromNow();
+
```
Comment (nihalanisumit) [inline client/src/utils/date/fromNow.js:4]: WOW that was easy.

**Review by nihalanisumit — APPROVED** [review]
LGTM

File: client/src/utils/date/fromNow.js (hunk 3 above)
Comment (piyushagwl007) [reply client/src/utils/date/fromNow.js:4]: Yes but the moment library is too big, need to remove the not used code when the build is made.

File: client/src/components/Comment/CommentComponent.jsx (hunk 1 above)
Comment (piyushagwl007) [reply client/src/components/Comment/CommentComponent.jsx:21]: Unless the comment is posted successfully in the backend. I need a temp Id to show adding and '...' for the posted on

**Pull request merged**